docs/dashboards/light-dark-theme.md

```diff
@@ -0,0 +1,16 @@
+Light and Dark Theme
+===
+
+The Light and Dark theme allows you to switch between a light and dark theme for the dashboard.  
+To use the Light and Dark theme, you need to import the `gui.css` file.
+
+```css
+    @import "https://code.highcharts.com/css/dashboards/gui.css";
+```
+
+By default, the theme is set to light. But it adjusts itself to the user's system settings.  
+You can also force the color scheme by setting the class to the dashboard container.
+
+```html
+    <div id="container" class="dashboards-dark ">
```

Comment:
All the other class names are namespaced as `highcharts-dashboards-*`. I think we should follow that convension. Requires `gui.css` to be updated.